src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+import { configure } from 'enzyme';
+
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
There is an `enzyme.config.js` file on the root folder which serves the same purpose for setting up the tests. The same file has been configured at the `jest.config.json`.

Reply:
done